src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -82,16 +81,14 @@ import { useBusinessStore, useConfigurationStore, useFilingHistoryListStore, use
     DownloadErrorDialog,
     FileCorrectionDialog,
     LoadCorrectionDialog
-  },
-  mixins: [FilingMixin]
+  }
 })
-export default class FilingHistoryList extends Vue {
+export default class FilingHistoryList extends Mixins(FilingMixin) {
   @Prop({ default: null }) readonly highlightId!: number
 
   @Getter(useFilingHistoryListStore) getCurrentFiling!: ApiFilingIF
   @Getter(useConfigurationStore) getEditUrl!: string
   @Getter(useFilingHistoryListStore) getFilings!: Array<ApiFilingIF>
-  @Getter(useBusinessStore) getIdentifier!: string
```

Comment:
Eslint was complaining since this getter was also in the mixin so I removed it. This applies to many places in the code.

Reply:
That's fine. I saw this (and changed it) too.

Reply:
(It's because the mixins are now properly "mixed into" `this`, so their method signatures are now recognized.)

Reply:
I'm OK to resolve this conversation.